Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -894,7 +894,7 @@ class BrowserViewController: UIViewController {
             self.webViewContainer.accessibilityElementsHidden = true
             UIAccessibility.post(notification: UIAccessibility.Notification.screenChanged, argument: nil)
         })
-        urlBar.locationView.reloadButton.reloadButtonState = .disabled
+        view.setNeedsUpdateConstraints()
```

Comment:
`view.setNeedsUpdateConstraints()` was moved to line 878. This should be removed.